vars/workflow-config-map-hidden.env

```diff
@@ -0,0 +1,8 @@
+workflowArtifactRepositoryConfigMapExecutorImage=argoproj/argoexec:v2.4.2
+workflowArtifactRepositoryConfigMapContainerRuntimeExecutor=docker
+workflowArtifactRepositoryConfigMapKeyFormat=artifacts/{{workflow.namespace}}/{{workflow.name}}/{{pod.name}}
+workflowArtifactRepositoryConfigMapAccessKeySecretName=onepanel
+workflowArtifactRepositoryConfigMapAccessKeySecretKey=argoArtifactRepositoryS3AccessKeyValue
```

Comment:
Does this have to be called `argoArtifactRepositoryS3AccessKeyValue`? The artifactRepository is not argo specific, other workflow providers can use it down the road. I noticed the secret key name changed in `onepanel` secret as well, again these are not Argo specific.